### ISSUE TITLE
group_concat not fully supported yet in 0.46

### ISSUE THREAD

**Issue by echou** [sqlparser-rs/sqlparser-rs#1269]
previous closed issue: #1225 

```sql
group_concat(
  DISTINCT a,b
  ORDER BY c, d
  SEPARATOR ';'
)
```

**SEPARATOR**  still cause error:

`ParserError("Expected ), found: SEPARATOR at Line: *, Column *`

**Comment by echou** [comment]
#1225

**Comment by echou** [comment]
i just found it was resolved and not in 0.46.
